pkgs/data/fonts/gelasio/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub }:
+
+fetchFromGitHub {
+  pname = "gelasio";
```

Comment:
You'll have to revert this to what we had before, since these aren't arguments `fetchFromGitHub` automatically accepts.